lib/services/component_manager.ts

```diff
@@ -223,6 +223,9 @@ export class SNComponentManager extends PureService {
           }
         }
         for (const component of syncedComponents) {
+          if (component.isEditor()) {
```

Comment:
Might it be worth a comment here? Kind of an abrupt condition without the proper context..